src/swagger/OpenAPIService.ts

```diff
@@ -233,9 +242,38 @@ export class OpenAPIService {
         return refs;
     }
 
+    /**
+     * @description Gets refs from allof schema only one level down
+     */
+    private getRefsByAllOf(
```

Comment:
looks like totally copy paste from getRefsByObject. Can you combine methods

Reply:
done